browser/app.css

```diff
@@ -937,4 +931,19 @@ footer {
     margin-right: 10px;
     float: right;
   }
+
+  .panel-group-items .panel:not(.zero-border):hover {
+    border-left: 3px solid #c00;
```

Comment:
border-left with 3px wide makes component card moves right 1px when mouse hovers over. We should ether have it 2px, or figure out how to make card stay on the same place when mouse hoovers over it.

Reply:
Fixed in https://github.com/redhat-developer-tooling/developer-platform-install/pull/1062/commits/6c04fe5849b2cc6e6cba9eb6257d1a40212922e4